### PR TITLE
feat(Roblox): Add images for largeImageKey + fixes

### PR DESCRIPTION
**Pull request by Snazzah**
Added profile avatars and group images to largeImageKey as a default off option. Also fixed bugs when trying to get the profile name and group name.

### CODE REVIEW

File: websites/R/Roblox/presence.ts [hunk 1]
```diff
@@ -33,14 +36,18 @@ presence.on("UpdateData", async () => {
   ) {
     profileName = <HTMLHeadingElement>(
       document.querySelector(
-        "div.profile-header-top > div.header-caption > div.header-title > h2"
+        "div.profile-header-top > div.header-caption div.header-title > h2"
```
Comment (Bas950) [inline websites/R/Roblox/presence.ts:39]: you sure this is correct?

Comment (Snazzah) [reply websites/R/Roblox/presence.ts:39]: Yeah, it worked for me. There was an extra div in between that was preventing this.

**Review by Bas950 — DISMISSED** [review]

File: websites/R/Roblox/presence.ts [hunk 2]
```diff
@@ -4,10 +4,12 @@ const presence = new Presence({
   browsingTimestamp = Math.floor(Date.now() / 1000);
 let profileName,
   profileTabs,
+  profileAvatar,
   messageTab,
   friendsTab,
   inventoryTab,
   groupName,
+  groupImage,
```
Comment (EncryptedDev) [inline websites/R/Roblox/presence.ts:12]: Why not `const`?

File: websites/R/Roblox/presence.ts [hunk 3]
```diff
@@ -4,10 +4,12 @@ const presence = new Presence({
   browsingTimestamp = Math.floor(Date.now() / 1000);
 let profileName,
   profileTabs,
+  profileAvatar,
```
Comment (EncryptedDev) [inline websites/R/Roblox/presence.ts:7]: Why not `const`?

File: websites/R/Roblox/presence.ts (hunk 2 above)
Comment (Snazzah) [reply websites/R/Roblox/presence.ts:12]: For the same reason the other variables aren't? I only matched the code/format style the file was already in.

File: websites/R/Roblox/presence.ts [hunk 4]
```diff
@@ -21,7 +23,8 @@ presence.on("UpdateData", async () => {
       document.querySelector(
         "div.game-calls-to-action > div.game-title-container > h2"
       )
-    );
+    ),
+    imagesEnabled = await presence.getSetting("images");
```
Comment (EncryptedDev) [inline websites/R/Roblox/presence.ts:27]:
```suggestion
    imagesEnabled = await presence.getSetting<boolean>("images");
```

File: websites/R/Roblox/presence.ts [hunk 5]
```diff
@@ -33,14 +36,18 @@ presence.on("UpdateData", async () => {
...
       )
     );
 
     profileTabs = <HTMLAnchorElement>(
       document.querySelector("#horizontal-tabs li.rbx-tab.active a")
     );
 
+    profileAvatar = <HTMLImageElement>(
+      document.querySelector(".avatar-headshot-lg thumbnail-2d img[image-load]")
+    );
```
Comment (EncryptedDev) [inline websites/R/Roblox/presence.ts:49]: Do this everywhere else too
```suggestion
    profileAvatar = document.querySelector<HTMLImageElement>(".avatar-headshot-lg thumbnail-2d img[image-load]");
```

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

**Review by EncryptedDev — CHANGES_REQUESTED** [review]
Do as I requested in my previous requested changed, define types properly

**Comment by Snazzah** [comment]
I committed the suggested changes, what other types are you talking about?

**Review by EncryptedDev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**